docs/usage/config-presets.md

```diff
@@ -5,23 +5,23 @@ description: Renovate's support for ESLint-like shareable configs
 
 # Shareable Config Presets
 
-This document describes how to configure your shared presets.
+This page describes how to configure your shared presets.
 Read the [Key concepts, presets](./key-concepts/presets.md) page to learn more about presets in general.
 
-Shareable config presets can only be used with the JSON format, other formats are not supported.
+Shareable config presets must use the JSON format, other formats are not supported.
```

Comment:
Need to check, we maybe support JSON5 or YAML already?

Reply:
we support JSON5